routers/web/web.go

```diff
@@ -60,6 +60,7 @@ func CorsHandler() func(next http.Handler) http.Handler {
 			AllowedOrigins: setting.CORSConfig.AllowDomain,
 			//setting.CORSConfig.AllowSubdomain // FIXME: the cors middleware needs allowSubdomain option
 			AllowedMethods:   setting.CORSConfig.Methods,
+			AllowedHeaders:   []string{"*"},
```

Comment:
Mostly we've just added the correct headers we need so in general I think this is a mistake.